### PR TITLE
ACC-1990 maxlength attribute on company limit

### PR DESCRIPTION
**Pull request by sakibshaik**
### Description
for some countries we want to restrict the company name to be maximum of 40 characters.so adding a max length attribute to the company name 

### Ticket
https://financialtimes.atlassian.net/browse/ACC-1990

### Reminder
Have you completed these common tasks (remove those that don't apply)?

- [y] **Documentation** updated or created
- [ y] **Tests** written for new or updated for existing functionality
- [ y] **Stories** updated to use this change
- [ y] **Product Review** ran past the product owner

### CODE REVIEW

File: components/company-name.jsx [hunk 1]
```diff
@@ -67,4 +69,5 @@ CompanyName.propTypes = {
 	fieldLabel: PropTypes.string,
 	isRequired: PropTypes.bool,
 	isHidden: PropTypes.bool,
+	maxlength: PropTypes.string,
```
Comment (ludrob) [inline components/company-name.jsx:72]: Should this be a number?
```suggestion
	maxlength: PropTypes.number,
```

**Review by jorbuedo — APPROVED** [review]

File: components/company-name.jsx [hunk 2]
```diff
@@ -37,6 +38,7 @@ export function CompanyName ({
 		required: isRequired,
 		disabled: isDisabled,
 		defaultValue: value,
+		maxlength,
```
Comment (ludrob) [inline components/company-name.jsx:41]: no default? Ok with me but in this case should proptype should tag this as required?
(And then should be released as major)

Comment (sakibshaik) [reply components/company-name.jsx:41]: it has been defaulted on line number 16 as a function param.

File: components/company-name.jsx [hunk 3]
```diff
@@ -13,6 +13,7 @@ export function CompanyName ({
 	placeholder = 'Please enter your company name',
 	isRequired = true,
 	isHidden = false,
+	maxlength = '',
```
Comment (ludrob) [inline components/company-name.jsx:16]: This should be a number.
```suggestion
	maxlength = 40,
```
?

Comment (sakibshaik) [reply components/company-name.jsx:16]: nope,i think  default can be  empty as i dont want to force the limit on all the consumers of this components

Comment (AniaMakes) [reply components/company-name.jsx:16]: I agree with Ludo on this. If Agne wants the limit to be 40, then there's a reason for this and we should be consistent across all of the consumers.

**Review by ludrob — APPROVED** [review]

File: components/company-name.jsx (hunk 3 above)
Comment (sakibshaik) [reply components/company-name.jsx]: that has been resolved by setting it default to 100

**Review by AniaMakes — APPROVED** [review]

**Comment by joejeet** [comment]
@sakibshaik Why 100 not 40 as the max length?

**Pull request merged**